pkg/openshift/openshift.go

```diff
@@ -42,3 +43,17 @@ func VerifyImageStreamExists(c *clients.Clients, name, namespace string) {
 		testsuit.T.Errorf("Failed to get image stream %q in namespace %q for tests: %s", name, namespace, err)
 	}
 }
+
+func IsCapabilityEnabled(c *clients.Clients, name string) bool {
+	log.Printf("Checking if OpenShift capability %s is enabled", name)
+
+	cv, err := c.ClusterVersion.Get(c.Ctx, "version", metav1.GetOptions{})
+	assert.NoError(err, "Could not get ClusterVersion instance\n")
```

Comment:
Will it show error when it fails?

Reply:
yes, it should fail with an error... it's the same as `testsuit.T.Errorf` on line 43 (I will unify it in a separate PR)